Removed link to developers page & updated link to binary packages (#58)
rizinorg/book: src/first_steps/getting_rizin.md

## Downloading rizin

Binary packages are available for most of the common platforms (e.g. Windows, MacOS, Linux), however you can still compile rizin yourself for many other architectures/operating systems.

> Please have a look at [releases](https://github.com/rizinorg/rizin/releases) in rizin repository for binary packages
> Please have a look at file [BUILDING.md](https://github.com/rizinorg/rizin/blob/dev/BUILDING.md) in Rizin repository for more detailed instructions on how to build the tool.

Reviewer: ```suggestion
Please have a look at the [releases](https://github.com/rizinorg/rizin/releases) tab of Rizin's GitHub repository for the binary packages.
See [BUILDING.md](https://github.com/rizinorg/rizin/blob/dev/BUILDING.md) for more detailed instructions on how to build the tool.
```
